extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html

```diff
@@ -227,8 +227,8 @@ <h2 class="state edit" ng-if="!editMode" ng-click="editState()">{{formattedState
 												</div>
 
 												<!-- default text output -->
-												<div layout="row" layout-align="start center" ng-show="!isOptionList(item)">
-													<div ng-show="isReadOnly(item)" layout="row" layout-align="start center" flex>
+												<div layout="row" layout-align="start center" ng-show="isReadOnly(item) || (!isReadOnyl(item) && !isOptionList(item))">
```

Comment:
Does this work for you? I assume it should not work because of the typo in the function name: `isReadOnyl`